crates/avalanche-consensus/src/snowman/snowball/binary/mod.rs

```diff
@@ -155,27 +155,27 @@ fn test_snowflake() {
     let snf = Snowflake::new(beta, red, 0, false);
     assert_eq!(snf.beta(), beta);
     assert_eq!(snf.preference(), red);
-    assert_eq!(snf.finalized(), false, "finalized too early");
+    assert!(!(snf.finalized()));
```

Comment:
Same comment everywhere. All the extra brackets are hard on the eyes and we probably shouldn't be dropping that message. 

```suggestion
    assert!(!snf.finalized(), "finalized too early");
```